lib/LoginStrategy.ts

```diff
@@ -351,6 +425,22 @@ export class LoginStrategy extends BaseStrategy {
     );
   }
 
+  /**
+   * extracts siwe message and signature either from request body or query
+   * @param req
+   * @returns {string} siwe object - message and signature
+   */
+  extractSiwe(req: Request): SiweReqPayload | null {
+    if (req.body.message && req.body.signature) {
```

Comment:
Looking at it again, I think having optional chaining would be useful here to avoid exceptions if the input is unexpected (e.g. if `req` doesn't have a `body` prop)
```suggestion
    if (req?.body?.message && req?.body?.signature) {
```